README.md

```diff
@@ -1 +1,221 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+
+  <h3><b>Leaderboard List App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Leaderboard]** is a a solo programming project that highlights the use of Webpack and ES6 features.
+
+## 🛠 Built With <a name="built-with">HTML CSS Javascript and Webpack</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  N/A
+</details>
+
+<details>
+  <summary>Server</summary>
+  Local
+</details>
+
+<details>
+<summary>Database</summary>
+ N/A
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Highlights the use of Webpack and ES6]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [Optional] [Highly Recommended] Please edit each section of your readme file by removing the guiding sentences and filling these sections with the correct data.

### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️

Reply:
Thank you for the feedback.